modules/auxiliary/gather/f5_bigip_cookie_disclosure.rb

```diff
@@ -148,5 +151,10 @@ def run
       )
     end
 
+    rescue ::Rex::ConnectionRefused, ::Rex::ConnectionTimeout, ::Errno::ECONNRESET, ::Rex::HostUnreachable, ::Timeout::Error, ::Errno::EPIPE
```

Comment:
`::Rex::ConnectionRefused` , `::Rex::ConnectionTimeout` , and `::Rex::HostUnreachable` are all encompassed (along with others) by `::Rex::ConnectionError`. Please use that instead.